### DISCUSSION TITLE
Geopandas 0.14.1 installed in conda environment but scripts still import 0.10.2

### DISCUSSION THREAD

**Discussion by dmgrogers** [geopandas/geopandas#3081] [Q&A]
Hi, I have installed version 0.14.1 of geopandas in a dedicated conda environment.  But, when I activate that environment and try to import geopandas, the version imported is 0.10.2 (geopandas.__version__), so some of the newer features aren't available.  Has anyone else had this issue?
![image](https://github.com/geopandas/geopandas/assets/46989040/90281afb-8bb5-4583-a7e7-9691cbc349c3)

**Comment by theroggy** [comment]
What does `conda list geopandas` return?
